addons/account/models/account_move_line.py

```diff
@@ -2028,7 +2028,7 @@ def _prepare_exchange_difference_move_vals(self, amounts_list, company=None, exc
             * to_reconcile: A list of tuple <move_line, sequence> in order to perform the reconciliation after the move
                             creation.
         """
-        company = self.company_id or company
+        company = company or self.company_id
```

Comment:
The docstring reads
> The company **in case there is no aml in self**

Would you mind giving an example of when it is needed to have something different than that?

Reply:
ping @ngochung207-viindoo